03-bash/task1.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+
+is_integer () {
+	local re='^[0-9]+$'
+	[[ $1 =~ $re ]]
+	return $?
+}
+
+integer_to_zero_string () {
+	local ret
+	printf -v ret "%04d" $1
+	echo "$ret"
+}
+
+is_ukr_locale() {
+	local re=$(locale | grep 'LANG=uk_UA')
+	if [ "$re" == '' ]; then
+		return 1
+	else
+		return 0
+	fi
+}
+
+exit_with_help () {
+	if is_ukr_locale; then
+		echo 'Usage_UKR:'
+		echo '  ./task1.sh [-h|--help] [-n num] [file]'
+		echo '     num --  amount of files'
+		echo '     file -- path to file to store results'
+	else
+		echo 'Usage:'
+		echo '  ./task1.sh [-h|--help] [-n num] [file]'
+		echo '     num --  amount of files'
+		echo '     file -- path to file to store results'
+	fi
+	exit $1
+}
+
+err_not_a_number() {
+	if is_ukr_locale; then
```

Comment:
multiple calls of is_ukr_locale.
It is better to save value as variable instead calling function.
Also it is better save localized test as variable.